Seq2SeqSharp/Applications/Options.cs

```diff
@@ -70,39 +80,50 @@ public class Options
         public bool EnableTensorCore = true;
 
         [Arg("The expert size for MoE model. Default is 1", nameof(ExpertNum))]
+        [Range(1, 8192)]
         public int ExpertNum = 1;
 
         [Arg("The factor of expert size per token for MoE model. Default is 1", nameof(ExpertsPerTokenFactor))]
+        [Range(1, 8192)]
         public int ExpertsPerTokenFactor = 1;
 
         [Arg("Maximum Segment Capacity. Default value is 16", nameof(MaxSegmentNum))]
+        [Range(1, 8192)]
         public int MaxSegmentNum = 16;
 
         [Arg("The network depth in encoder.", nameof(EncoderLayerDepth))]
+        [Range(1, 8192)]
         public int EncoderLayerDepth = 1;
 
         [Arg("The embedding dim in source side", nameof(SrcEmbeddingDim))]
+        [Range(1, 8192)]
         public int SrcEmbeddingDim = 128;
 
         [Arg("Starting Learning rate factor for encoders", nameof(EncoderStartLearningRateFactor))]
+        [Range(0.000000001f, 1.0f)]
```

Comment:
It could be 0 if users want to freeze the layers, so the range should be [0.0, 1.0]